src/cowrie/commands/curl.py

```diff
@@ -19,77 +17,7 @@
 
 commands = {}
 
-
-class command_curl(HoneyPotCommand):
-    """
-    curl command
-    """
-    limit_size = CowrieConfig().getint('honeypot', 'download_limit_size', fallback=0)
-    download_path = CowrieConfig().get('honeypot', 'download_path')
-
-    def start(self):
-        try:
-            optlist, args = getopt.getopt(self.args, 'sho:O', ['help', 'manual', 'silent'])
-        except getopt.GetoptError as err:
-            # TODO: should be 'unknown' instead of 'not recognized'
-            self.write("curl: {}\n".format(err))
-            self.write("curl: try 'curl --help' or 'curl --manual' for more information\n")
-            self.exit()
-            return
-
-        for opt in optlist:
-            if opt[0] == '-h' or opt[0] == '--help':
-                self.curl_help()
-                return
-            elif opt[0] == '-s' or opt[0] == '--silent':
-                self.silent = True
-
-        if len(args):
-            if args[0] is not None:
-                url = str(args[0]).strip()
-        else:
-            self.write("curl: try 'curl --help' or 'curl --manual' for more information\n")
-            self.exit()
-            return
-
-        if '://' not in url:
-            url = 'http://' + url
-        urldata = compat.urllib_parse.urlparse(url)
-
-        outfile = None
-        for opt in optlist:
-            if opt[0] == '-o':
-                outfile = opt[1]
-            if opt[0] == '-O':
-                outfile = urldata.path.split('/')[-1]
-                if outfile is None or not len(outfile.strip()) or not urldata.path.count('/'):
-                    self.write('curl: Remote file name has no length!\n')
-                    self.exit()
-                    return
-
-        if outfile:
-            outfile = self.fs.resolve_path(outfile, self.protocol.cwd)
-            path = os.path.dirname(outfile)
-            if not path or \
-                    not self.fs.exists(path) or \
-                    not self.fs.isdir(path):
-                self.write('curl: %s: Cannot open: No such file or directory\n' % outfile)
-                self.exit()
-                return
-
-        url = url.encode('ascii')
-        self.url = url
-
-        self.artifactFile = Artifact(outfile)
-        # HTTPDownloader will close() the file object so need to preserve the name
-
-        self.deferred = self.download(url, outfile, self.artifactFile)
-        if self.deferred:
-            self.deferred.addCallback(self.success, outfile)
-            self.deferred.addErrback(self.error, url)
-
-    def curl_help(self):
-        self.write("""Usage: curl [options...] <url>
+CURL_HELP = """Usage: curl [options...] <url>
```

Comment:
This was a mess. A huge help string inside the code. I removed it from the class source and put it to a constant before the class.